README.md

```diff
@@ -131,6 +136,55 @@ To invalidate tags, call the method `Psr6Store::invalidateTags` or use the
 `PurgeTagsListener` from the [FOSHttpCache][3] library to handle tag 
 invalidation requests.
 
+### Expiry of content digest cache items
+
+To optimize storage, responses that share the same
+content also share the same cache item for this content.
+E.g. the very same HTML response is never cached twice
+but rather referenced to. The cache item for the request
+itself shall be called "request meta cache item" and the
+content it references to, is the "content digest cache item".
+You can find the calculated hash in the "X-Content-Digest"
+response header (prefixed with "X-" for compatibility with
+the Symfony default Store implementation).
+
+When a request meta cache item gets invalidated,
+we cannot invalidate  the content digest cache item because
+that would mean, it implicitly invalidates all the other
+request meta cache items, as their content digest cache item
+is not available anymore.
+However, never deleting them at all would mean they remain in
+our cache forever.
+This is kind of a bad situation because the content digest is
+usually a lot bigger than the meta information. So it's the
+one we would really like to clean up!
+
+To solve this, we need to also expire content digest cache items.
+We cannot, however, expire them the same time we expire the request
+meta cache item (based it's Cache-Control or Expire headers).
+Why not? Imagine this:
+
+- Request 1:
+  The response is a 10 MB file and is cacheable for 10 minutes.
+- Request 2:
+  The response is the same 10 MB file but in this case, it's
+  cacheable for 24 hours.
+
+
+If we were to expire the content digest item the same as the meta
+cache item, it would mean it will expire after 10 minutes and thus,
+implicitly also expire our cache item for request 2 although we
+could've kept that in the cache for 24 hours!
```

Comment:
I think the example is wrong. The content digest cache item is overwritten when the second request is processed, therefore it will expire 24 hours past the second request?

Reply:
I'll remove all those docs anyway and will reword them then. Will check thx.